apps/backoffice-v2/src/lib/blocks/variants/DefaultBlocks/hooks/useDefaultBlocksLogic/useDefaultBlocksLogic.tsx

```diff
@@ -232,7 +234,7 @@ export const useDefaultBlocksLogic = () => {
   });
 
   const entityInfoBlock = useEntityInfoBlock({
-    entity: workflow?.context?.entity,
+    entity: workflow?.context?.entity ?? {},
```

Comment:
An empty object would make the data checks harder as opposed to undefined.

Reply:
Fair, so it's just a TS error right now (as `useentityInfoBlock` expects `entity` to not be undefined)